elixir/apps/web/lib/web/live/gateway_groups/new.ex

```diff
@@ -45,34 +45,30 @@ defmodule Web.GatewayGroups.New do
           </.form>
 
           <div :if={not is_nil(@group)}>
+            <div class="mb-4">
+              <div class="text-xl mb-4">
+                Gateway Instance Group created!
+              </div>
+              <div class="text-gray-600 mb-2">
+                The token below is used to connect Gateways to this group. <br />
+              </div>
+              <.code_block id="gateway-token" class="w-full rounded mb-2" phx-no-format><%= encode_group_token(@group) %></.code_block>
+              <div class="text-gray-600">
+                This token will not be shown again. Please copy the token and store it in a secure location.
```

Comment:
I don't think that we need to ask to store it, just should warn that it will be not shown again. We don't want users to store the tokens in 1password or anything like that (they can always create a new group).

Reply:
cc @jefferenced

I think users are going to store this in 1Pass, otherwise it's a major pain to add new instances to this set which I think will be somewhat common for users to do.

Creating a new group is a lot of friction because it requires going back and editing all existing resources to link to the new group

Reply:
@jamilbk let's discuss a use cases and maybe come up with a good UX for it, but not promote saving it anywhere